Dynatrace/dynatrace-operator
# Allow to use empty proxy password

Clients use proxies with empty proxy password. A CR is accepted by the proxy_url validator if proxy password is empty.

How can this be tested?
Use empty password in the Proxy Url (proxy.value, proxy secret) and try to apply your CR - should be accepted.

## Checklist
